tango/common/aliases.py

```diff
@@ -12,6 +12,7 @@ class EnvVarNames(Enum):
     CLICK_LOGGER_ENABLED_ENV_VAR = "TANGO_CLICK_LOGGER_ENABLED"
     LOGGING_HOST_ENV_VAR = "TANGO_LOGGING_HOST"
     LOGGING_PORT_ENV_VAR = "TANGO_LOGGING_PORT"
+    LOGGING_PREFIX = "LOGGING_PREFIX"
```

Comment:
For consistency with the other env var names and to make this more "tango-specific", I think we should prefix by `TANGO_`:
```suggestion
    LOGGING_PREFIX = "TANGO_LOGGING_PREFIX"
```